client/src/app/videos/+video-watch/video-watch.component.ts

```diff
@@ -470,6 +477,10 @@ export class VideoWatchComponent implements OnInit, OnDestroy {
 
       this.player.on('customError', ({ err }: { err: any }) => this.handleError(err))
 
+      if(this.user && this.user.autoPlayNextVideo){
```

Comment:
put this in a `else if` after https://github.com/Chocobozzz/PeerTube/pull/2137/files#diff-b7342a12f863e2ea4fe04b8f4f704fbfR489